tests/helm/goldenfile_test.go

```diff
@@ -132,11 +133,173 @@ func runGoldenFileTest(t *testing.T, valuesFileName string, outputFileName strin
 // expected templates
 func fixupRenderedYaml(yaml string, chartVersion string) string {
 	checksumRegex := regexp.MustCompile("checksum/config: [a-z0-9]{64}")
-
+	// replacements := map[string]string{
+	// 	fmt.Sprintf("app.kubernetes.io/version: \"%s\"", chartVersion): "app.kubernetes.io/version: \"%CURRENT_CHART_VERSION%\"",
+	// 	fmt.Sprintf("chart: \"sumologic-%s\"", chartVersion):           "chart: \"sumologic-%CURRENT_CHART_VERSION%\"",
+	// 	fmt.Sprintf("chart: sumologic-%s", chartVersion):               "chart: sumologic-%CURRENT_CHART_VERSION%",
+	// 	fmt.Sprintf("client: k8s_%s", chartVersion):                    "client: k8s_%CURRENT_CHART_VERSION%",
+	// 	fmt.Sprintf("value: \"%s\"", chartVersion):                     "value: \"%CURRENT_CHART_VERSION%\"",
+	// }
+	replacements := []string{
+		"app.kubernetes.io/version: \"%s\"",
+		"chart: \"sumologic-%s\"",
+		"chart: sumologic-%s",
+		"client: k8s_%s",
+		"value: \"%s\"",
```

Comment:
Use %q instead of %s for items that need to be surrounded in quotes, as this allows you to avoid escapes here.

Reply:
addressed